documentation/cobigen-intellij-installation.asciidoc

```diff
@@ -0,0 +1,67 @@
+:toc:
+toc::[]
+
+
+= Installation
+
+
+== IntelliJ setup
+
+If you want to use IntelliJ for CobiGen project, you can follow the steps below:
+
+. You can install and create script for IntelliJ as described in
+
+`https://github.com/devonfw/ide/blob/master/documentation/intellij.asciidoc`
+
+. After starting IntelliJ, You **need** to import the cobigen folder as maven project
+
+. You now **need** to configure maven settings in IntelliJ as follows
```

Comment:
You need to configure maven settings in IntelliJ now or Next you have to configure the maven settings in IntelliJ as follow or just leave the now